core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -436,12 +436,24 @@ oppia.directive('conversationSkin', [
             }
 
             if (ExplorationPlayerStateService.isStateTerminal(stateName)) {
-              ExplorationRecommendationsService.getRecommendedSummaryDicts(
-                ExplorationPlayerStateService.getAuthorRecommendedExpIds(
-                  stateName),
-                function(summaries) {
-                  $scope.recommendedExplorationSummaries = summaries;
-                });
+              $scope.parentExplorationIds = UrlService.getParamValuesAsList(
+                'parent');
+              if ($scope.parentExplorationIds) {
```

Comment:
Check for array length instead (see comment below).

Reply:
Done